graphql/appointment/mutations.ts

```diff
@@ -113,7 +117,9 @@ export class MutateAppointmentResolver {
     async appointmentDecline(@Ctx() context: GraphQLContext, @Arg('appointmentId') appointmentId: number) {
         const appointment = await getLecture(appointmentId);
         await hasAccess(context, 'Lecture', appointment);
-        await declineAppointment(context.user, appointment);
+        const subcourse = await prisma.subcourse.findUnique({ where: { id: appointment.subcourseId }, include: { course: true } });
```

Comment:
What about match appointments? Shouldn't this only be silent if appointment type == course?